src/main/java/sirius/search/suggestion/Suggest.java

```diff
@@ -215,18 +233,49 @@ private SearchRequestBuilder generateRequestBuilder(SuggestBuilder phraseSuggest
      * @return tuples of the query string and all suggest options
      */
     private List<PhraseSuggestion.Entry.Option> execute(SearchRequestBuilder sqb) {
+        this.future = sqb.execute();
+        return getSuggestions();
+    }
 
-        SearchResponse response = sqb.execute().actionGet();
-        PhraseSuggestion phraseSuggestion = response.getSuggest().getSuggestion("suggestPhrase");
+    /**
+     * In contrast to {@link #execute(SearchRequestBuilder)} this is executed async. The retrieved suggestions can be
+     * retrieved using {@link #getSuggestions()}.
+     *
+     * @return the helper itself for fluent method calls
+     */
+    public Suggest<E> executeAsync() {
+        this.future = generateRequestBuilder(generateSuggestionBuilder()).execute();
```

Comment:
returning the suggest-object itself has the advantage that we can use the helper method "getSuggestions()" which extracts the suggest options from the future